lib/TPP/Transforms/CombineXsmmPass.cpp

```diff
@@ -146,14 +102,17 @@ struct CombineXsmmOp : public OpRewritePattern<xsmm::BrgemmOp> {
     IntegerType integer64 = IntegerType::get(rewriter.getContext(), 64);
 
     Location loc = brgemmOp.getLoc();
-    auto dims = getSizesAndLeadingDimForBrgemmOp(rewriter, brgemmOp);
+    auto dims = DenseI64ArrayAttr::get(
```

Comment:
The old code was doing validation of the input. Are we sure that the brgemm dispatch will always be valid for the fused brgemm?

Reply:
The old was broken (it recomputed and values changed parameters, e.g. stride), it's unclear why it was there and being used. In this case we just need to copy from BRGEMM to fusedBRGEMM. So this patch is fix.